.circleci/config.yml

```diff
@@ -238,6 +263,11 @@ workflows:
           filters:
             tags:
               only: /.*/
+        # ping @g-k / gguthe on slack with feedback
+      - rust-changelog:
```

Comment:
Could put this in a separate workflow too if that's preferable.

Reply:
actually, is there a more generic address that could be used? I assume you want to take vacations or possibly do other things in the future, maybe take on a minion you can assign stuff like this too, etc.